### ISSUE TITLE
loader not placed in center of the screen

### ISSUE THREAD

**Issue by ccchen1991** [sebgroup/ng-components#446]
**Describe the bug**
Loader should show in center of the screen like previous `ac-loader` behave.

**To Reproduce**
1. Define the component like this:
```
<sebng-loader [toggle]="true"></sebng-loader>
```
2. Realise the loader not show in center of the screen.
3. Workaround for this is to add in some styling to make it center.
```
<sebng-loader class="m-3 d-flex justify-content-center" [toggle]="true"></sebng-loader>
```
**Expected behavior**
Same center position like `ac-loader`

**Screenshots**
![image](https://user-images.githubusercontent.com/34128411/88769916-a1aa5c80-d1af-11ea-854b-6e0f9e0b5512.png)

**Additional context**
after inspect on the css, maybe is because missing the wrapper class like `ac-loader`
![image](https://user-images.githubusercontent.com/34128411/88770175-fd74e580-d1af-11ea-9422-dc5797597b32.png)

**Comment by yousifalraheem** [comment]
Thank you for submitting your first issue. We will look into it as soon as we can. If you want to address this issue yourself, please read our [contributing guide](https://github.com/sebgroup/react-components/blob/develop/CONTRIBUTING.md) and submit a pull request.

**Comment by ccchen1991** [comment]
kindly update for the next release. 🎉 😉

**Comment by yousifalraheem** [comment]
fixed in #447
